### DISCUSSION TITLE
3.2.1-release 部署问题，启动就报错，读配置文件有问题

### DISCUSSION THREAD

**Discussion by Carl7713** [apache/dolphinscheduler#15672] [Q&A]
Error creating bean with name 'masterServer': Unsatisfied dependency expressed through field 'masterRegistryClient'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'masterRegistryClient': Unsatisfied dependency expressed through field 'failoverService'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'failoverService' defined in file [D:\IdeaProjects\dolphinscheduler3.2.1\dolphinscheduler\dolphinscheduler-master\target\classes\org\apache\dolphinscheduler\server\master\service\FailoverService.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'masterFailoverService' defined in file [D:\IdeaProjects\dolphinscheduler3.2.1\dolphinscheduler\dolphinscheduler-master\target\classes\org\apache\dolphinscheduler\server\master\service\MasterFailoverService.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'registryClient' defined in file [D:\IdeaProjects\dolphinscheduler3.2.1\dolphinscheduler\dolphinscheduler-registry\dolphinscheduler-registry-api\target\classes\org\apache\dolphinscheduler\registry\api\RegistryClient.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.apache.dolphinscheduler.registry.api.Registry' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}

**Comment by liuzhao900928** [comment]
我也遇到了相同问题，3.2.2版本部署集群模式时候有这样的问题
![dolphinschedular1](https://github.com/user-attachments/assets/94bf3259-d1a0-449c-b6e8-1bf649b1e476)
![dolphinschedular2](https://github.com/user-attachments/assets/7baa6ae0-e468-4d55-aa43-1be389a190d5)
